src/js/index.js

```diff
@@ -16,6 +18,25 @@ var reparse = require('./actions/vegaInvalidate');
 var model = require('./model');
 model.init();
 
+// Set up the listeners that connect the model to the store
+var listeners = require('./store/listeners');
+
+// Bind the listener that will flow changes from the redux store into Vega
+store.subscribe(listeners.createStoreListener(
+  store,
+  model,
+  listeners.recreateVegaIfNecessary,
```

Comment:
are these new instances or could you reference these methods from inside the file instead of passing them?